standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/RetryingMetaStoreClient.java

```diff
@@ -333,4 +352,9 @@ private boolean hasConnectionLifeTimeReached(Method method) {
     return shouldReconnect;
   }
 
+  @VisibleForTesting
+  public int getRetriesMade(){
+    return this.retriesMade;
+  }
+
 }
```

Comment:
please undo this `retriesMade` to field variable conversion; 
instead you could probably try to:
* check the exception being thrown - if that should be different
* or introduce a specialized `HiveMetaStoreClient` in your test